towhee/tests/mock_operators/norm_data_operator/norm_data_operator.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import NamedTuple
+from towhee.operator import Operator, SharedType
```

Comment:
`SharedType` is unused - this will not pass `pylint`.

Reply:
Thanks, I will delete `SharedType`.